apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Tests/RetryHandlerTest.cs

```diff
@@ -16,38 +16,37 @@
 using System.Net.Http;
 using Xunit;
 
-namespace Google.Cloud.Storage.V1.Tests
```

Comment:
We'll need more actual tests here to make sure the retry options are actually honoured though.

Reply:
"Hide whitespace" under settings can help in this case.

Reply:
I meant just for the ease of review - not ignoring the white space. Should have been clear. The formatting needs to be fixed.

Reply:
The formatting *is* okay in the long run - the problem is that the same commit is making changes to formatting (moving the test to use a file-scoped namespace, which is a positive change in itself) *and* making functional changes.
I agree that reviewing without whitespace would be a workaround - but then it would hide other changes that actually need to be fixed. I think it's cleaner to separate out the different kinds of change in different commits, so that we can review all commits with whitespace *shown* in order to see any issues, but without detracting from the review.

Reply:
I have raised another PR for these changes as suggested